users/views.py

```diff
@@ -1,3 +1,50 @@
-from django.shortcuts import render
+import json
+import bcrypt
+import jwt
+import re
 
-# Create your views here.
+from django.views                        import View
+from django.http                           import HttpResponse, JsonResponse
+from .models                                 import Grade, Job, User, Gender
+
+id_valid = r'^(?=.*[a-z])[a-z0-9]{6,20}$'
+
+class IdVerificationView(View) :
+        def post(self, request) :
+
+                user_id  = json.loads(request.body)
+                if User.objects.filter(account = user_id['account']).exists():
+                        return JsonResponse({'message' : 'id_repetition'}, status = 400)
+
+                if not re.match(id_valid, user_id['account']):
+                        return JsonResponse({'message' : 'unproper_id'}, status = 400)
```

Comment:
indentation 4칸으로 조정하고 INVALID_ID 가 어떨까요?
```suggestion
                        return JsonResponse({'message' : 'INVALID_ID'}, status = 400)
```